### PR TITLE
Swagger Documentation of ChargesApiResource.java

### PR DESCRIPTION
**Pull request by luckyman20**
GCI Task: 6575931940405248

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/charge/api/ChargesApiResource.java [hunk 1]
```diff
@@ -141,7 +144,10 @@ public String createCharge(final String apiRequestBodyAsJson) {
     @Path("{chargeId}")
     @Consumes({ MediaType.APPLICATION_JSON })
     @Produces({ MediaType.APPLICATION_JSON })
-    public String updateCharge(@PathParam("chargeId") final Long chargeId, final String apiRequestBodyAsJson) {
+    @ApiOperation(value = "Update a Charge", httpMethod = "PUT", notes = "Updates the details of a Charge.")
+    @ApiImplicitParams({@ApiImplicitParam(value = "body", required = true, paramType = "body", dataType = "body", format = "body", dataTypeClass = ChargesApiResourceSwagger.PutChargesRequest.class)})
```
Comment (Sanyam96) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/charge/api/ChargesApiResource.java]: Refractor the name of inner class to "PutChargesChargeIdRequest"

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/charge/api/ChargesApiResource.java [hunk 2]
```diff
@@ -141,7 +144,10 @@ public String createCharge(final String apiRequestBodyAsJson) {
...
+    @ApiResponses({@ApiResponse(code = 200, message = "OK", response = ChargesApiResourceSwagger.PutChargesResponse.class)})
```
Comment (Sanyam96) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/charge/api/ChargesApiResource.java]: Refractor the name of inner class to "PutChargesChargeIdResponse"

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/charge/api/ChargesApiResource.java [hunk 3]
```diff
@@ -154,7 +160,9 @@ public String updateCharge(@PathParam("chargeId") final Long chargeId, final Str
     @Path("{chargeId}")
     @Consumes({ MediaType.APPLICATION_JSON })
     @Produces({ MediaType.APPLICATION_JSON })
-    public String deleteCharge(@PathParam("chargeId") final Long chargeId) {
+    @ApiOperation(value = "Delete a Charge", httpMethod = "DELETE", notes = "Deletes a Charge.")
+    @ApiResponses({@ApiResponse(code = 200, message = "OK", response = ChargesApiResourceSwagger.DeleteChargesResponse.class)})
```
Comment (Sanyam96) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/charge/api/ChargesApiResource.java]: Refractor the name of inner class to "DeleteChargesChargeIdResponse"

**Comment by luckyman20** [comment]
@Sanyam96 Updated!!

**Pull request merged**